### DISCUSSION TITLE
Descriptions and examples with database and retention policy for Enterprise 1.x forward compatibility

### DISCUSSION THREAD

**Discussion by jstirnaman** [influxdata/openapi#532] [General]
@lesam @davidby-influx Should we add request and code samples to the relevant bucket endpoints for https://github.com/influxdata/docs-v2/pull/4416? For example, https://github.com/influxdata/openapi/blob/500ecc44a57a976e49a102d9bd3ca9e856e6bc3d/src/common/paths/buckets_bucketID.yml#L56?

@kelseiv @lwandzura @sunbryely-influxdata

**Comment by davidby-influx** [comment]
@jstirnaman - I think so, but we have to be careful to document the limited subset of commands supported in 1.X and call out the ones that don't work. there

**Comment by jstirnaman** [comment]
_Originally posted by @davidby-influx in https://github.com/influxdata/docs-v2/pull/4416#discussion_r965398396_:

> I think we should include an example of the predicate in the `delete` API.  I know it is covered in the V2 API docs, but it might be useful to discuss the de`delete` body with `start`, `stop`, and `predicate`

> There are restrictions on the predicate.
> 
> > delete - predicate only supports equality operators and conjunctions
